shell/client/accounts/account-settings.html

```diff
@@ -139,6 +144,22 @@ <h3 class="title-bar">Primary e-mail address for service-related notifications</
   </div>
 </template>
 
+<template name="accountSuspended">
+  <h1>
+    This account has been suspended by you.
+  </h1>
+
+  <h2>
+    {{#if timeUntilDeletion}}
+     This account will be deleted in {{timeUntilDeletion}}. Would you like to cancel deletion and restore this account?
+     {{else}}
+     This account is suspended and all of its grains and shares are disabled. Would you like to restore this account?
```

Comment:
Only the admin can suspend an account without a deletion time, right? But the heading on this page says "this account has been suspended by you"?